specification/07-theory_operations.adoc

```diff
@@ -893,6 +893,15 @@ compromising any TVM confidential assets.
 As the platform resources owner, the host supervisor domain manager can assign
 a TDI to a TVM by binding them together (step 3). At any point in time, it can
 reclaim that physical resource by unbinding (step 8a) it from its TVM.
+Graceful and explicit unbinding through the `COVH` ABI will clear all
+confidential data and abort any outstanding DMA requests, as described in the
+xref:interface-unbinding[Interface Unbinding] section. Implicit TDI unbinding,
+i.e. attempts from the host supervisor domain manager to reclaim and remap a
+TDI without explicitly notifying the corresponding TSM, represents a security
+threat that is in the
+xref:05-security_model.adoc#security-model[CoVE-IO threat model] scope. In that
+situation, both the DSM and the TSM must protect the confidentiality and
+integrity of the TVM assets.
```

Comment:
any ref/example on how that is done - e.g. on subsequent use of the TDI, the DSM would enforce it would transition into the error state; also what about in-flight transactions in that case?